credit.py

```diff
@@ -1,30 +1,36 @@
-import os, sys, logging
-import json
+import os, sys, json, logging
 from werkzeug.security import generate_password_hash, check_password_hash
 
 
-logging.basicConfig(filename="main.log", 
-                    format='%(asctime)s %(message)s', 
-                    filemode='w') 
-logger = logging.getLogger() 
-logger.setLevel(logging.ERROR) 
+logger = logging.getLogger('main.py')
```

Comment:
Please use `__file__` instead of `'main.py'`

Reply:
But it doesn't work.

Reply:
Could you describe what goes wrong?